suite-native/atoms/src/AlertBox.tsx

```diff
@@ -46,44 +41,51 @@ const textStyle = prepareNativeStyle(_ => ({
 
 const variantToColorMap = {
     info: {
-        backgroundColor: 'backgroundAlertBlueSubtleOnElevation0',
+        backgroundColor: 'backgroundAlertBlueSubtleOnElevation1',
         contentColor: 'iconAlertBlue',
+        borderColor: 'backgroundAlertBlueSubtleOnElevationNegative',
     },
     success: {
-        backgroundColor: 'backgroundPrimarySubtleOnElevation0',
+        backgroundColor: 'backgroundPrimarySubtleOnElevation1',
         contentColor: 'textSecondaryHighlight',
+        borderColor: 'backgroundPrimarySubtleOnElevationNegative',
     },
     warning: {
-        backgroundColor: 'backgroundAlertYellowSubtleOnElevation0',
-        contentColor: 'textAlertYellow',
+        backgroundColor: 'backgroundAlertYellowSubtleOnElevation1',
+        contentColor: 'iconAlertYellow',
+        borderColor: 'backgroundAlertYellowSubtleOnElevationNegative',
     },
     error: {
-        backgroundColor: 'backgroundAlertRedSubtleOnElevation0',
+        backgroundColor: 'backgroundAlertRedSubtleOnElevation1',
         contentColor: 'textAlertRed',
+        borderColor: 'backgroundAlertRedSubtleOnElevationNegative',
     },
 } as const satisfies Record<AlertBoxVariant, AlertBoxStyle>;
 
 const variantToIconName = {
     info: 'info',
     success: 'checkCircle',
     warning: 'warningTriangle',
-    error: 'warningTriangle',
+    error: 'warningCircle',
 } as const satisfies Record<AlertBoxVariant, IconName>;
 
 export type AlertBoxProps = {
     variant: AlertBoxVariant;
     title: ReactNode;
-    isStandalone?: boolean;
+    borderRadius?: number;
 };
 
-export const AlertBox = ({ title, isStandalone = false, variant = 'info' }: AlertBoxProps) => {
+export const AlertBox = ({ title, variant = 'info', borderRadius }: AlertBoxProps) => {
```

Comment:
nit: If there is a default value, let's assign it already in the prop declarations:
```suggestion
export const AlertBox = ({ title, variant = 'info', borderRadius = nativeBorders.radii.medium}: AlertBoxProps) => {
```

Reply:
good spot! 
b9d6388345